loklak/loklak_webclient
# Analysis of Followers Fixes #312

Fixes #312 
How to Test?
Type in the screenname of the user you want to see folowers of.Currenlty its analysing 2000 followers only.Please give it sometime to analyse after clicking.
![image](https://cloud.githubusercontent.com/assets/2105062/9003044/37272328-3788-11e5-900f-37c2ef99a72b.png)
